### PR TITLE
Upgrade to OpenVR 1.4.18

### PR DESCRIPTION
**Pull request by schulzch**
This PR depends on [openvr-sys#10](https://github.com/rust-openvr/rust-openvr-sys/pull/10) and does:

- upgrade to OpenVR 1.4.18 (most recent)
- fix some typing issues related to enums
- fix a warning related to `ATOMIC_BOOL_INIT`

As said in the other PR, I am willing to do maintenance for a while, as more PRs towards camera support are likely to follow.

### CODE REVIEW

**Comment by schulzch** [comment]
Nice and tidy commits (similar to the other PR). CI will fail as `openvr-sys` `2.0.3` is not published yet. Otherwise it should be mergeable.

File: src/compositor/mod.rs [hunk 1]
```diff
@@ -151,7 +151,7 @@ impl Compositor {
     }
 }
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Copy, Clone)]
```
Comment (Ralith) [inline src/compositor/mod.rs]: Why no `Debug`?

Comment (schulzch) [reply src/compositor/mod.rs]: Because `[tracking::TrackedDevicePose; 64]` does not implement Debug - not sure why this has not been an issue before.

Comment (Ralith) [reply src/compositor/mod.rs]: Ah, it used to be 16.

**Comment by Ralith** [comment]
rustc 1.35 is giving me this warning:
```
warning: borrow of packed field is unsafe and requires unsafe function or block (error E0133)
  --> src/system/event.rs:18:70
   |
18 |             event: Event::from_sys(x.eventType as sys::EVREventType, &x.data),
   |                                                                      ^^^^^^^
   |
   = note: #[warn(safe_packed_borrows)] on by default
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #46043 <https://github.com/rust-lang/rust/issues/46043>
   = note: fields of packed structs might be misaligned: dereferencing a misaligned pointer or even just creating a misaligned reference is undefined behavior
```

**Comment by schulzch** [comment]
That one is weird. `rustc 1.35.0 (3c235d560 2019-05-20)` gives me, when adding an unsafe:
```
warning: unnecessary `unsafe` block
  --> src\system\event.rs:18:70
   |
18 |             event: Event::from_sys(x.eventType as sys::EVREventType, unsafe {&x.data}),
   |                                                                      ^^^^^^ unnecessary `unsafe` block
   |
   = note: #[warn(unused_unsafe)] on by default
```

**Comment by Ralith** [comment]
That makes sense, since you're compiling on windows where the struct isn't packed. `#[allow(unused_unsafe)]` should do the trick.

**Comment by schulzch** [comment]
Done

**Comment by schulzch** [comment]
Anything missing? :)

**Comment by Ralith** [comment]
Looks good, thanks!

**Pull request merged**